### DISCUSSION TITLE
Export/Import function for room list

### DISCUSSION THREAD

**Discussion by dacoex** [element-hq/element-meta#770] [Ideas]
#### Is your suggestion related to a problem? Please describe.
I wanted to remove / leave some room but keep their address for later.
This I wanted to expert a backup of my currently subscribed rooms to a external file.
This is currently not possible.

#### Describe the solution you'd like.
Under settings, there should be a button with import/expert functionality:
Export/Import
* Export room list
* Import list

#### Describe alternatives you've considered.
manually writing down rooms :-(

#### Additional context
None.

**Comment by DantrazTrev** [comment]
An option could be placed in the three-dot menu for the rooms. At least for exporting?
![Element-3-dante-s-test-dum](https://user-images.githubusercontent.com/43457420/116504997-57d5da80-a8d7-11eb-9203-966646610d2b.png)

This could be much simpler in exporting as the room list is directly accessible within the component?

**Comment by DantrazTrev** [comment]
Not the exact implementation of what the author might be looking for.
But I made the export part that copies the Room Names and Room Ids from the context menu.

![ezgif com-gif-maker (3)](https://user-images.githubusercontent.com/43457420/117535252-80b24a00-b012-11eb-9b1a-2c8b394fbdab.gif)
